### DISCUSSION TITLE
Are wildcard domains supported with nginx mode?

### DISCUSSION THREAD

**Discussion by begunfx** [acmesh-official/acme.sh#4644] [Q&A]
I'm trying to setup nginx proxy server, but I've run into a snag.  I ran the following command to copy the certs from acme.sh to the ngix custom_ssl folder:

acme.sh --install-cert -d example.com \
--key-file       /volume1/docker/npm/data/custom_ssl/key.pem  \
--fullchain-file /volume1/docker/npm/data/custom_ssl/cert.pem \
--reloadcmd     "service nginx force-reload"

Hopefully this is correct.

Then I'm trying to re-issue the certificate using nginx mode, but I keep getting an error:
[Sat May 27 13:54:12 PDT 2023] entry
[Sat May 27 13:54:12 PDT 2023] Error, can not get domain token entry *.exampledomain.com for http-01
[Sat May 27 13:54:12 PDT 2023] The supported validation types are: dns-01 , but you specified: http-01
[Sat May 27 13:54:12 PDT 2023] pid
[Sat May 27 13:54:12 PDT 2023] No need to restore nginx, skip.
[Sat May 27 13:54:12 PDT 2023] _clearupdns
[Sat May 27 13:54:12 PDT 2023] dns_entries
[Sat May 27 13:54:12 PDT 2023] skip dns.
[Sat May 27 13:54:12 PDT 2023] _on_issue_err
[Sat May 27 13:54:12 PDT 2023] Please check log file for more details: /usr/local/share/acme.sh/acme.sh.log
[Sat May 27 13:54:12 PDT 2023] _chk_vlist
[Sat May 27 13:54:12 PDT 2023] 'nginx:' does not contain 'dns'
[Sat May 27 13:54:12 PDT 2023] socat doesn't exist.
[Sat May 27 13:54:12 PDT 2023] Diagnosis versions: 

command run:
acme.sh --renew --nginx -d exampledomain.com -d '*.exampledomain.com' --force

I'm running the latest version of acme.sh (v3.0.6)

Any ideas?

Thanks!

**Comment by neilpang** [comment]
no.

wildcard cert can only be validated by dns mode.
